src/style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+.input {
+  display: block;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300;
+  font-size: 16px;
+  font-family: inherit;
+}
+
+.input-small {
+  height: 42px;
+  width: 100%;
+  margin-top: 20px;
+  padding-left: 33px;
+  font-family: inherit;
+  font-size: 14px;
+}
+
+.vector {
+  display: block;
+  position: absolute;
+}
+
+.vector-small {
+  height: 11px;
+  width: 11px;
+  margin-left: 13px;
+  margin-top: 15px;
+}
+
+.vector-big {
+  width: 19px;
```

Comment:
you can move all rules to general selector .vector and only use modificator for small input